runtime/runtime.go

```diff
@@ -300,6 +322,10 @@ func (rt *Runtime) Serve(ctx context.Context) error {
 		}
 	}
 
+	if !rt.Params.Incognito {
+		go rt.checkOPAUpdateLoop(ctx, defaultUploadIntervalSec, nil)
```

Comment:
Question: do we need to hookup the `done` channel for clean shutdown?

Reply:
Added a done channel to the runtime struct to shutdown the report loop.